routes/signupRouter.js

```diff
@@ -0,0 +1,6 @@
+const userController = require('../controllers/userController');
```

Comment:
I would absolutely not create a separate router for this. This belongs in the userRouter instead.

Reply:
we will fix it with you in our branch then